executor/executor_test.go

```diff
@@ -3442,29 +3443,23 @@ type testSuiteWithCliBaseCharset struct {
 }
 
 func (s *testSuiteWithCliBaseCharset) SetUpSuite(c *C) {
-	collate.SetNewCollationEnabledForTest(true)
-	collate.SetCharsetFeatEnabledForTest(true)
 	s.testSuiteWithCliBase.SetUpSuite(c)
 }
 
 func (s *testSuiteWithCliBaseCharset) TearDownSuite(c *C) {
 	s.testSuiteWithCliBase.TearDownSuite(c)
-	collate.SetNewCollationEnabledForTest(false)
-	collate.SetCharsetFeatEnabledForTest(false)
 }
 
 type testSuiteWithCliBaseCharsetNoNewCollation struct {
```

Comment:
Maybe we can remove this testcase? It looks same as `testSuiteWithCliBaseCharset`